packages/core/content-platform/base.jsonc

```diff
@@ -0,0 +1,349 @@
+                "maxItems": {
+                  "title": "Maximum Number of History Items",
+                  "type": "integer",
+                  "default": 5
+                }
+              }
+            },
+            "searchTop": {
+              "title": "Top Search",
+              "type": "object",
+              "properties": {
+                "title": {
+                  "title": "Title",
+                  "type": "string",
+                  "default": "Top Search"
+                },
+                "maxItems": {
+                  "title": "Maximum Number of Top Search Items",
+                  "type": "integer",
+                  "default": 5
+                }
+              }
+            },
+            "searchAutocomplete": {
+              "title": "Autocomplete",
+              "type": "object",
+              "properties": {
+                "icon": {
+                  "title": "Icon",
+                  "type": "object",
+                  "properties": {
+                    "icon": {
+                      "title": "Icon",
+                      "type": "string",
+                      "enumNames": ["Magnifying Glass"],
+                      "enum": ["MagnifyingGlass"]
+                    },
+                    "alt": {
+                      "type": "string",
+                      "title": "Alternative Label",
+                      "default": "Magnifying Glass"
+                    }
+                  }
+                },
+                "maxItems": {
+                  "title": "Maximum Number of Autocomplete Items",
+                  "type": "integer",
+                  "default": 5
+                }
+              }
+            },
+            "searchProducts": {
+              "title": "Suggested Products",
+              "type": "object",
+              "properties": {
+                "title": {
+                  "title": "Title",
+                  "type": "string",
+                  "default": "Suggested Products"
+                },
+                "maxItems": {
+                  "title": "Maximum Number of Suggested Products",
+                  "type": "integer",
+                  "default": 5
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "content-types": {
+    "globalSections": {
+      "title": "Global Sections",
+      "$singleton": true,
+      "$extends": ["#/components/base-page-template"]
+    },
+    "globalHeaderSections": {
+      "title": "Global Header Sections",
+      "$singleton": true,
+      "$extends": ["#/components/base-page-template"]
+    },
+    "globalFooterSections": {
+      "title": "Global Footer Sections",
+      "$singleton": true,
+      "$extends": ["#/components/base-page-template"]
+    },
+    "pdp": {
+      "title": "Product Page",
+      "$extends": ["#/components/base-page-template"],
+      "properties": {
+        "template": {
+          "$ref": "#/components/core.template"
+        },
```

Comment:
I think we can remove this part because "template" is not a valid component for Faststore. I remember we added here as an example.